src/index.jsx

```diff
@@ -77,9 +77,26 @@ export default class UIIdentificationCard extends React.Component {
       transition: 'all .2s ease-in-out',
       textAlign: 'left',
     },
-    path: {
-      transition: 'all .2s ease-in-out',
+    rect: {
+      borderRadius: '8px',
+      boxShadow: `0 2px 2px rgba(0, 0, 0, 0.2),
+        0 0 40px rgba(0, 0, 0, 0.1)`
     },
+    specular: {
+      position: 'absolute',
+      top: '0px',
+      left: '0px',
+      backgroundSize: '100% 100%',
+      borderRadius: '8px',
+      transition: 'all .2s ease-in-out',
+      WebkitMask: '-webkit-linear-gradient(top, #fff, transparent 80%)',
+      WebkitFilter: 'opacity(0.5)',
```

Comment:
Could you please add a comment to list why using `filter.opacity` instead of the pure `opacity` property, like the below:

```js
// use -webkit-filter's filter function will enable the hardware acceleration for rendering perf than the `opacity` property.
```